From 991ba354eec26b849c52ff7386d13baabde64569 Mon Sep 17 00:00:00 2001
From: Jamie Heilman <jamie@audible.transient.net>
Date: Sat, 16 Jun 2012 05:10:45 +0000
Subject: [PATCH 3/3] documentation: fix some typos and misleading comments

MOUNT_NFS_DEFAULT_PROTOCOL *is* used by more than just replicated
server maps on modern systems.
---
 man/auto.master.5.in           |    3 +--
 modules/replicated.c           |    1 -
 redhat/autofs.sysconfig.in     |    7 ++-----
 samples/autofs.conf.default.in |    7 ++-----
 4 files changed, 5 insertions(+), 13 deletions(-)

diff --git a/man/auto.master.5.in b/man/auto.master.5.in
index fff9943..3a0ddb3 100644
--- a/man/auto.master.5.in
+++ b/man/auto.master.5.in
@@ -207,8 +207,7 @@ Maps are browsable by default (program default "yes").
 .B MOUNT_NFS_DEFAULT_PROTOCOL
 Specify the default protocol used by mount.nfs(8) (program default 3). Since
 we can't identify this default automatically we need to set it in the autofs
-configuration. This option will only make a difference for replicated map
-entries as availability probing isn't used for single host map entries.
+configuration.
 .TP
 .B APPEND_OPTIONS
 Determine whether global options, given on the command line or per mount
diff --git a/modules/replicated.c b/modules/replicated.c
index 5a79a72..614e74d 100644
--- a/modules/replicated.c
+++ b/modules/replicated.c
@@ -512,7 +512,6 @@ static unsigned int get_nfs_info(unsigned logopt, struct host *host,
 		      "called for host %s proto %s version 0x%x",
 		      host->name, proto, version);
 
-	/* Try to prode UDP first to conserve socket space */
 	rpc_info->proto = getprotobyname(proto);
 	if (!rpc_info->proto)
 		return 0;
diff --git a/redhat/autofs.sysconfig.in b/redhat/autofs.sysconfig.in
index a46335d..36b924d 100644
--- a/redhat/autofs.sysconfig.in
+++ b/redhat/autofs.sysconfig.in
@@ -14,7 +14,7 @@ TIMEOUT=300
 #
 #NEGATIVE_TIMEOUT=60
 #
-# MOUNT_WAIT - time to wait for a response from umount(8).
+# MOUNT_WAIT - time to wait for a response from mount(8).
 # 	       Setting this timeout can cause problems when
 # 	       mount would otherwise wait for a server that
 # 	       is temporarily unavailable, such as when it's
@@ -34,10 +34,7 @@ BROWSE_MODE="no"
 # MOUNT_NFS_DEFAULT_PROTOCOL - specify the default protocol used by
 # 			       mount.nfs(8). Since we can't identify
 # 			       the default automatically we need to
-# 			       set it in our configuration. This will
-# 			       only make a difference for replicated
-# 			       map entries as availability probing isn't
-# 			       used for single host map entries.
+# 			       set it in our configuration.
 #
 #MOUNT_NFS_DEFAULT_PROTOCOL=3
 MOUNT_NFS_DEFAULT_PROTOCOL=4
diff --git a/samples/autofs.conf.default.in b/samples/autofs.conf.default.in
index b87c4d0..ac2f63b 100644
--- a/samples/autofs.conf.default.in
+++ b/samples/autofs.conf.default.in
@@ -14,7 +14,7 @@ TIMEOUT=300
 #
 #NEGATIVE_TIMEOUT=60
 #
-# MOUNT_WAIT - time to wait for a response from umount(8).
+# MOUNT_WAIT - time to wait for a response from mount(8).
 # 	       Setting this timeout can cause problems when
 # 	       mount would otherwise wait for a server that
 # 	       is temporarily unavailable, such as when it's
@@ -34,10 +34,7 @@ BROWSE_MODE="no"
 # MOUNT_NFS_DEFAULT_PROTOCOL - specify the default protocol used by
 # 			       mount.nfs(8). Since we can't identify
 # 			       the default automatically we need to
-# 			       set it in our configuration. This will
-# 			       only make a difference for replicated
-# 			       map entries as availability probing isn't
-# 			       used for single host map entries.
+# 			       set it in our configuration.
 #
 #MOUNT_NFS_DEFAULT_PROTOCOL=3
 #
-- 
1.7.10

